### PR TITLE
Elevating PipelineDriver logs to debug level

### PR DESCRIPTION
**Pull request by kwen2501**
Porting changes made by Huaqing.

Currently PipelineDriver logs are at the info level, which makes the training log too verbose. Elevating the PipelineDriver logging to "debug" level.

### CODE REVIEW

**Comment by XilunWu** [comment]
Hi Ke, I have the same `pyre check` error in my PR as this one in yours:
https://github.com/pytorch/tau/actions/runs/3120890274/jobs/5061824857#step:5:54

Do we need to wait for someone to fix that to pass this check?

**Comment by kwen2501** [comment]
> 

If it wasn't caused by your PR then no need to wait.

File: pippy/PipelineDriver.py [hunk 1]
```diff
@@ -1301,7 +1301,7 @@ def run_until(self, predicate: Callable[[pippy.fx.Node], bool]) -> Optional[pipp
 
     def run_one(self, node):
         # TODO: hoist run() implementation
-        logging.info(f'[{self.cur_microbatch}] Issue command to run {node.format_node()}')
+        logging.debug(f'[{self.cur_microbatch}] Issue command to run {node.format_node()}')
```
Comment (XilunWu) [inline pippy/PipelineDriver.py:1304]: Issue -> issuing to keep a consistency maybe?

**Review by XilunWu — APPROVED** [review]
LGTM besides we can double check if there's any polishing can be done to the print info. Thx for making the logging output more readable @kwen2501 !

File: pippy/PipelineDriver.py (hunk 1 above)
Comment (kwen2501) [reply pippy/PipelineDriver.py:1304]: Will fix in new PRs.

**Pull request merged**